### PR TITLE
Data view

### PR DESCRIPTION
**Pull request by KyleWalks**
Add graphs/table to data tab. Fix styling.

### CODE REVIEW

File: birdspotter/birdspotter/dataviz/views.py [hunk 1]
```diff
@@ -1,7 +1,55 @@
...
+# Data view
 def index(request):
-	return HttpResponse("This is the data index")
\ No newline at end of file
+    args = {}
+
+    # Total bird count data
+    birds = ["Gull", "Razorbill", "Common Eider", "Sandpiper"
+             , "Cormorants", "Great Cormorant"]
+    data_total = {'Specie': [birds[i] for i in range(len(birds))],
```
Comment (NickKania) [inline birdspotter/birdspotter/dataviz/views.py:17]: Can this just be `'Specie': birds`?

**Review by NickKania — APPROVED** [review]
This is a good POC, we should talk with the client to see what graphs and visualizations are useful to them and other scientists and adjust accordingly.

File: birdspotter/birdspotter/dataviz/views.py (hunk 1 above)
Comment (KyleWalks) [reply birdspotter/birdspotter/dataviz/views.py:17]: It creates an error. "arrays must all be same length"

Comment (KyleWalks) [reply birdspotter/birdspotter/dataviz/views.py]: Okay, I fixed it. Should look better now.

**Pull request merged**